apps/src/simpleSignUp/lti/registration/LtiDynamicRegistrationPage.tsx

```diff
@@ -37,6 +41,11 @@ export const LtiDynamicRegistrationPage = ({
         // Send post message to Canvas parent window
         // https://canvas.instructure.com/doc/api/file.registration.html#registration-response
         window.parent.postMessage({subject: 'org.imsglobal.lti.close'}, '*');
+        analyticsReporter.sendEvent(
```

Comment:
I had to move this event to the front-end, as the view runs outside of the application layout, so application.rb doesn't set `statsig_stable_id` as a cookie. This now relies on StatsigReporter.js to set the cookie if it's not already set.